tests/unit/presigned_get_object_test.py

```
@@ -63,3 +65,35 @@ def test_can_include_response_headers(self):
+            bucket_name='mybucket',
+            object_name='myfile.pdf',
+            region='us-west-2',
+        )
+        self.assertIn('%2Fus-west-2%2Fs3%2Faws4_request', url)
+
+    def test_region_conflicts_with_client_region(self):
+        """Test the passed region is validated against the client region."""
+        client = Minio(
+            endpoint='localhost:9000',
+            access_key='my_access_key',
+            secret_key='my_secret_key',
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "secret_key"

(S106)